File: ReservationServiceClient/model/reservationservice/base/VehicleInFirm.cs

public class VehicleInFirm {
    public long id { get; set; }
    public long firm_id { get; set; }
    public long vehicle_id { get; set; }
    public Double price { get; set; }
    public long reservation_id { get; set; }
    public VehicleInFirm() {
    }

    public VehicleInFirm(long id, long firm_id, long vehicle_id, double price, long reservation_id) {
        this.id = id;
        this.firm_id = firm_id;
        this.vehicle_id = vehicle_id;
        this.price = price;
        this.reservation_id = reservation_id;
    }

    
}

File: reservation-service/src/main/java/rs/raf/reservationservice/domain/VehicleInFirm.java
package rs.raf.reservationservice.domain;

import javax.persistence.*;
import lombok.*;

@Entity
@Table(name = "vehicle_in_firm")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class VehicleInFirm {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @OneToOne
    private Firm firm;
    @OneToOne
    private Vehicle vehicle;
    @Column(name = "price")
    private Double price;
}